tests/container_test.py

```diff
@@ -5,7 +5,19 @@
 import os
 import time
 
+<<<<<<< HEAD
 READY_MESSAGE = "Starting nginx"
+=======
+# Third-Party Libraries
+import pytest
+
+ENV_VAR = "ECHO_MESSAGE"
+ENV_VAR_VAL = "Hello World from docker compose!"
+READY_MESSAGE = "This is a debug message"
+SECRET_QUOTE = (
+    "There are no secrets better kept than the secrets everybody guesses."  # nosec
+)
+>>>>>>> 9cea64da4a1379cdde3c509a09fc548f0da8b034
```

Comment:
You have committed a merge conflict.  Please resolve.

Reply:
@jsf9k resolved with [d38e133](https://github.com/cisagov/domain-manager-ui/pull/247/commits/d38e1335f747335baa33f87d2745c9cf0cc84808)